### PR TITLE
Update stashServiceMock.ts

### PR DESCRIPTION
**Pull request by MariuszSzpyt**
Upgrading stash service mock so its easy to maintain + moving microapps PR tests to chops:

- PR tests are now executed mostly in chops (for swap and deposit)
- New stash mock - it will pass most requests to real stash service except /xcm/channels to point it to chops env
- axios and https packages added for calls to true stash service
- new STASH_URL env variable to store "true" stash service address
- tests under microapps-ui for deposit and swap are skipped for now - they will be needed though once we wanna run tests for runtime upgrade scenario

### CODE REVIEW

File: test/microapps-ui-xcm/microapps-ui-main.deposit.test.ts [hunk 1]
```diff
@@ -134,11 +134,11 @@ describe("Microapps UI deposit modal tests - no action", () => {
     await depositModal.selectToken(KSM_ASSET_NAME);
     await depositModal.enterValue("1");
 
-    // await depositModal.waitForContinueState(true, 180000);
+    await depositModal.waitForContinueState(true, 180000);
```
Comment (goncer) [inline test/microapps-ui-xcm/microapps-ui-main.deposit.test.ts:137]: :O 180s?

File: test/microapps-ui-xcm/microapps-ui-main.swap.test.ts [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    await swap.setPayTokenAmount("10001");
+
+    const areTradeDetailsDisplayed = await swap.areTradeDetailsDisplayed();
+    expect(areTradeDetailsDisplayed).toBeTruthy();
+
+    const swapFee = await swap.fetchSwapFee();
+    expect(swapFee).toEqual(0);
+
+    const isSwapFeeAlert = await swap.isSwapFeeAlert();
+    expect(isSwapFeeAlert).toBeFalsy();
```
Comment (goncer) [inline test/microapps-ui-xcm/microapps-ui-main.swap.test.ts:195]: alert is false in both cases?

File: test/microapps-ui-xcm/microapps-ui-main.swap.test.ts [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    await sleep(10000000);
+
+    const areTradeDetailsDisplayed = await swap.areTradeDetailsDisplayed();
+    expect(areTradeDetailsDisplayed).toBeTruthy();
+
+    const swapFee = await swap.fetchSwapFee();
+    expect(swapFee).toEqual(50);
+
+    const isSwapFeeAlert = await swap.isSwapFeeAlert();
+    expect(isSwapFeeAlert).toBeTruthy();
```
Comment (goncer) [inline test/microapps-ui-xcm/microapps-ui-main.swap.test.ts:234]: ahh, is that alert :)

**Review by goncer — APPROVED** [review]

File: test/microapps-ui-xcm/microapps-ui-main.deposit.test.ts (hunk 1 above)
Comment (MariuszSzpyt) [reply test/microapps-ui-xcm/microapps-ui-main.deposit.test.ts:137]: ah yea leftover from debug, not need to be that long , will fix!

**Pull request merged**